wiki/People/osu!monthly_Team/en.md

```diff
@@ -0,0 +1,40 @@
+# osu!monthly Team
+
+The **osu!monthly Team** manages the [osu!monthly](/wiki/Community/osu!monthly) newsletter.
+
+The contributors of [osu!monthly](/wiki/Community/osu!monthly) are active players in the community, keeping track of what is going on each month and curating the content to be included in each issue.
+
+## Team members
+
+### Main contributors
+
+The people listed below are the core of the osu!monthly Team and participate every month in the production of the news post.
```

Comment:
not sure but that reads odd otherwise
```suggestion
The people listed below are the core of the osu!monthly Team and participate every month in the production of the news posts.
```